### PR TITLE
docs: Update teams.rst

### PR DESCRIPTION
**Pull request by ro4i7**
##### SUMMARY

solved: #14520
solved under hacktoberfest 2023

##### ISSUE TYPE

 - Bug, Docs Fix or other nominal change

##### COMPONENT NAME

 - API
 - UI
 - Collection
 - CLI
 - Docs
 - Other

##### AWX VERSION

```

```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Comment by ro4i7** [comment]
@tvo318 please check this

File: docs/docsite/rst/userguide/teams.rst [hunk 1]
```diff
@@ -14,10 +14,12 @@ Access the Teams page by clicking **Teams** from the left navigation bar. The te
 
 
 .. image:: ../common/images/organizations-teams-list.png
+   :alt: Teams Page
```
Comment (tvo318) [inline docs/docsite/rst/userguide/teams.rst]: Can we be more descriptive? maybe something helpful to someone with limited sight would be, "Teams page containing a list of teams and the organizations they belong to."

File: docs/docsite/rst/userguide/teams.rst [hunk 2]
```diff
@@ -60,6 +64,7 @@ This tab displays the list of Users that are members of this Team. This list may
 |Teams - users list|
 
 .. |Teams - users list| image:: ../common/images/teams-users-list.png
+   :alt: Users List
```
Comment (tvo318) [inline docs/docsite/rst/userguide/teams.rst]: I suggest something more descriptive along these lines:
"Teams list showing the Access tab displaying a list of users and their roles."

File: docs/docsite/rst/userguide/teams.rst [hunk 3]
```diff
@@ -78,10 +83,12 @@ In order to add a user to a team, the user must already be created. Refer to :re
 To remove roles for a particular user, click the disassociate (x) button next to its resource.
 
 .. image:: ../common/images/permissions-disassociate.png
+   :alt: Disassociate Button
```
Comment (tvo318) [inline docs/docsite/rst/userguide/teams.rst]: I suggest something more descriptive along these lines:
"Access tab with list of users and an arrow pointing to the disassociate button next to a user's role."

File: docs/docsite/rst/userguide/teams.rst [hunk 4]
```diff
@@ -97,6 +104,7 @@ Selecting the **Roles** view displays a list of the permissions that are current
 |Teams - permissions list|
 
 .. |Teams - permissions list| image:: ../common/images/teams-permissions-sample-roles.png
+   :alt: Permissions List
```
Comment (tvo318) [inline docs/docsite/rst/userguide/teams.rst]: I suggest something more descriptive along these lines:
"Permissions list with resource names, type and their associated roles."

File: docs/docsite/rst/userguide/teams.rst [hunk 5]
```diff
@@ -118,26 +126,22 @@ To add permissions to a Team:
 3. Click to select the resource to assign team roles and click **Next**.
 
 .. image:: ../common/images/teams-permissions-templates-select.png
+   :alt: Permissions Templates Select
```
Comment (tvo318) [inline docs/docsite/rst/userguide/teams.rst]: This implies it is just for templates. I suggest something like:
"Add Teams Permissions Wizard step 2, choose the resources from the list."

File: docs/docsite/rst/userguide/teams.rst [hunk 6]
```diff
@@ -118,26 +126,22 @@ To add permissions to a Team:
...
 
 4. Click the checkbox beside the role to assign that role to your chosen type of resource. Different resources have different options available.
 
 .. image:: ../common/images/teams-permissions-template-roles.png
+   :alt: Permissions Template Roles
```
Comment (tvo318) [inline docs/docsite/rst/userguide/teams.rst]: I suggest something like:
"Add Teams Permissions Wizard step 3, choose the roles to apply to the previously selected resource."

File: docs/docsite/rst/userguide/teams.rst [hunk 7]
```diff
@@ -97,6 +104,7 @@ Selecting the **Roles** view displays a list of the permissions that are current
 |Teams - permissions list|
 
 .. |Teams - permissions list| image:: ../common/images/teams-permissions-sample-roles.png
+   :alt: Permissions List
```
Comment (tvo318) [inline docs/docsite/rst/userguide/teams.rst:108]: Missing image after Step 1: teams-users-add-permissions-form.png. 
I suggest something like:
"Add Teams Permissions Wizard step 1, choose the resource type."

File: docs/docsite/rst/userguide/teams.rst [hunk 8]
```diff
@@ -118,26 +126,22 @@ To add permissions to a Team:
...
 .. image:: ../common/images/teams-permissions-templates-select.png
+   :alt: Permissions Templates Select
 
 4. Click the checkbox beside the role to assign that role to your chosen type of resource. Different resources have different options available.
 
 .. image:: ../common/images/teams-permissions-template-roles.png
+   :alt: Permissions Template Roles
 
 
 5. Click **Save** when done, and the Add Permissions Wizard closes to display the updated profile for the user with the roles assigned for each selected resource.
```
Comment (tvo318) [inline docs/docsite/rst/userguide/teams.rst]: Found a typo in original text - Should be "...updated profile for the team with the roles assigned..."

File: docs/docsite/rst/userguide/teams.rst [hunk 9]
```diff
@@ -118,26 +126,22 @@ To add permissions to a Team:
...
 
 .. image:: ../common/images/teams-permissions-sample-roles.png
-  
+   :alt: Updated Profile
```
Comment (tvo318) [inline docs/docsite/rst/userguide/teams.rst]: I suggest something like:
"Updated profile for each team's resources and their roles."

**Review by tvo318 — CHANGES_REQUESTED** [review]
Suggested a few alt text revisions... Also, please amend your final commit to do the DCO signoff per the [Contributor's Guide guidelines](https://ansible.readthedocs.io/projects/awx/en/latest/contributor/work_items.html#things-to-know-prior-to-submitting-revisions)? To do this:
1. Do a git commit --signoff
2. Then push the new commits to your branch

Thanks again for your valuable contributions!

**Comment by tvo318** [comment]
@ro4i7 - can you address these changes, please? If not I'll update your PR with my own changes. thanks!

**Review by tvo318 — APPROVED** [review]
Thanks @ro4i7 for your contributions. I've added my own commits to yours and went ahead and approved it!

**Pull request merged**